spec/models/document_spec.rb

```diff
@@ -10,8 +10,9 @@
   let(:link) { nil }
   let(:chunking_method) { :basic }
   let(:chunking_profile) { create(:chunking_profile, method: chunking_method, size: 800) }
+  let(:parent) { nil }
 
-  describe "when document has a web link" do
+  describe "with a web link" do
```

Comment:
I might try refactoring this spec so we follow the `describe "#methodname"` convention, but for now you could just change all the `describe` calls to `context`. They read more like `context`s.

Reply:
For now I'll switch to `context`.

Reply:
Reading more of this spec, I feel like just using `context` will be fine as long as it's possible to come up with example names that read well and make the linter happy.

I sometimes run into an issue with RSpec where I want to describe some behaviour but I need to call multiple methods of the subject to describe the behaviour. The `describe "#methodname` pattern gets in the way of this. So I think for some classes - maybe all models - we could skip `describe` blocks and go straight to `context`.